bloom-housing/bloom
# Update select labels

# Pull Request Template

## Issue

Addresses #1761


## Description

Updates select options to use translations.

## Type of change


## How Can This Be Tested/Reviewed?

Open the "Add unit" form and check the "Unit Type" field options. They should be translated.


## Checklist:
